Referenced by issue #59340, Python custom Operators fail to Undo properly if executed while in edit mode
